src/xdr_ioq.c

```diff
@@ -327,6 +329,13 @@ xdr_ioq_uv_advance(struct xdr_ioq *xioq)
 	len = ioquv_length(uv);
 	xioq->ioq_uv.plength += len;
 
+#if 0
```

Comment:
I thought about removing it completely, it was really helpful during some of the debug, but also very wordy for some stuff that uses 2k buffers. One thought would be to put it under a separate flag so it could be turned off (maybe there could be a buffer details flag)?